### PR TITLE
Add jungle minions killed

### PR DESCRIPTION
**Pull request by DarkIntaqt**
Nothing major, just new fields. Tests all passed

### CODE REVIEW

**Comment by TheDrone7** [comment]
Could this be used to fine-tune the `creepScore` property?
https://github.com/DarkIntaqt/shieldbow/blob/ft-jungle/src/structures/api/Participant.ts#L665

Still never know what cs numbers are right and what are wrong.

**Comment by DarkIntaqt** [comment]
The CS calculated are correct the way they are. `totalAllyJungleMinionsKilled` and `totalEnemyJungleMinionsKilled` added together should be the same as `neutralMinionsKilled`. If not, im confused but also not amazed. 
Anyways, the CS calculation should already be fine

File: src/structures/api/Participant.ts [hunk 1]
```diff
@@ -588,6 +596,9 @@ export class Participant {
       mitigated: data.damageSelfMitigated
     };
 
+    this.totalAllyJungleMinionsKilled = data.totalAllyJungleMinionsKilled;
+    this.totalEnemyJungleMinionsKilled = data.totalEnemyJungleMinionsKilled;
```
Comment (TheDrone7) [inline src/structures/api/Participant.ts:600]: Could this be used to fine-tune the `creepScore` property? Still never know what CS numbers are right.

https://github.com/TheDrone7/shieldbow/blob/42eb3c1d4fc678dcad7acfbc9f926d4e2d2a450a/src/structures/api/Participant.ts#L654

**Review by TheDrone7 — APPROVED** [review]

**Review by TheDrone7 — APPROVED** [review]
Looks good for the most part but the tests seem to be failing, might need to increase the timeout for the champion mastery tests.

**Pull request merged**